caluma/caluma_core/jexl.py

```diff
@@ -233,3 +229,21 @@ def visit_Transform(self, transform):
                         yield arg.value
 
         yield from self.generic_visit(transform)
+
+
+class ExtractTransformSubjectAndArgumentsAnalyzer(CalumaAnalyzer):
+    """
+    Extract all referenced subjects and arguments of a given transforms.
+
+    If no transforms are given all references of all transforms will be extracted.
+    """
+
+    def visit_Transform(self, transform):
+        if not self.transforms or transform.name in self.transforms:
+            if not isinstance(transform.subject, type(transform)):
+                yield (
+                    transform.subject.value,
+                    [arg for arg in transform.args if isinstance(arg, Literal)],
```

Comment:
What's the reason behind only returning literal arguments? Do we even have cases where that wouldn't be true?

Reply:
No idea, this was taken from `ExtractTransformArgumentAnalyzer`. I can remove it if you want.